hockeysdk/src/main/java/net/hockeyapp/android/utils/BoundedPrintWriter.java

```diff
@@ -0,0 +1,27 @@
+package net.hockeyapp.android.utils;
+
+import java.io.PrintWriter;
+import java.io.Writer;
+
+public class BoundedPrintWriter extends PrintWriter {
```

Comment:
You should extend it from just `Writer` because `PrintWriter` has other `write` overloads which don't call this overridden one.

Reply:
printStackTrace expects PrintWriter not Writer